src/schedule/ll_schedule.c

```diff
@@ -175,6 +175,9 @@ static void schedule_ll_tasks_run(void *data)
 	if (schedule_ll_is_pending(sch))
 		schedule_ll_tasks_execute(sch, last_tick);
 
+	notifier_event(sch, NOTIFIER_ID_LL_POST_RUN,
```

Comment:
Seems that Notifier API is entered from LL context while clients may be pre-empted in the middle of their register/unregister calls, so the Notifier context data access should be probably locally synchronized.

Reply:
Done.